.github/workflows/on-push.yml

```diff
@@ -18,6 +18,9 @@ jobs:
         with:
           node-version: ${{ steps.nvm.outputs.NVMRC }}
 
+      - name: Enable Corepack
+        run: corepack enable
```

Comment:
Why do we need this?

Reply:
To install yarn 4. Otherwise it will install yarn 1.22.2.

> error This project's package.json defines "packageManager": "yarn@4.1.0". However the current global version of Yarn is 1.22.2